### PR TITLE
FAT-1459  Verify the possibility to modify MARC Bibliographic record

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

**Comment by TetianaParanich** [comment]
I mistakenly deleted this pr

File: cypress/fixtures/modifyMarcBib.mrc [hunk 1]
```diff
@@ -0,0 +1 @@
...
  atwo-dimensional moving image
  aEU-Staaten
  aEU-Verkehrspolitik
  aEuropäische Union
  aHochgeschwindigkeitsverkehr
  aHochgeschwindigkeitszug
  aSchienenverkehr
  aVerkehrsnetz
2 aEuropäische Kommission Generaldirektion Mobilität und Verkehr
ffi1b74ab75-9f41-4837-8662-a1d99118008d
```
Comment (dmitriyfolio) [inline cypress/fixtures/modifyMarcBib.mrc:1]: Is this file related with existing file cypress\fixtures\oneMarcBib.mrc?

File: cypress/integration/data-import/modify-marcBib-record.spec.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      },
+      existingRecordFields: {
+        field: '001',
+      },
+      matchCriterion: 'Exactly matches',
+      existingRecordType: 'MARC_BIBLIOGRAPHIC'
+    };
+
+    settingsDataImport.goToMatchProfiles();
+    matchProfiles.createMatchProfile(matchProfile);
```
Comment (dmitriyfolio) [inline cypress/integration/data-import/modify-marcBib-record.spec.js:78]: steps 12 and 16 related with 2 match profiles creation. You test has jut one step with match profile creation. Please check your design/sync testrail and the cypress code

File: cypress/support/fragments/data_import/mapping_profiles/fieldMappingProfiles.js [hunk 3]
```diff
@@ -57,6 +57,14 @@ export default {
...
 
+  createModifyMappingProfile:(mappingProfile) => {
+    openNewMappingProfileForm();
+    newMappingProfile.fillModifyMappingProfile(mappingProfile);
+    closeViewModeForMappingProfile();
+    closeViewModeForMappingProfile();
+    cy.expect(actionsButton.exists());
+  },
+
   checkMappingProfilePresented: (mappingProfile) => {
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/fieldMappingProfiles.js:68]: Done

File: cypress/integration/data-import/modify-marcBib-record.spec.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/modify-marcBib-record.spec.js:78]: It must be job profile in step 16. I fixed it in the test case.

**Review by dmitriyfolio — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

**Pull request merged**